qemu.go

```diff
@@ -217,32 +217,46 @@ func (q *qemu) appendFSDevices(devices []ciaoQemu.Device, podConfig PodConfig) [
 	return devices
 }
 
-func (q *qemu) appendConsoles(devices []ciaoQemu.Device, podConfig PodConfig) []ciaoQemu.Device {
+func (q *qemu) appendConsoles(devices []ciaoQemu.Device, podConfig *PodConfig) []ciaoQemu.Device {
 	serial := ciaoQemu.SerialDevice{
 		Driver: ciaoQemu.VirtioSerial,
 		ID:     "serial0",
 	}
 
 	devices = append(devices, serial)
 
+	if podConfig.Console == "" {
+		podConfig.Console = fmt.Sprintf("%s/%s/console.sock", runStoragePath, podConfig.ID)
```

Comment:
There are quite a few references to `console.sock` now. I'd consider creating a `const` for it to avoid repetition.